src/pcsswebservice/pom.xml

```diff
@@ -99,7 +99,11 @@
 							<sourceRoot>${project.build.directory}/generated-sources/cxf</sourceRoot>
 							<wsdlOptions>
 								<wsdlOption>
+									<wsdl>${basedir}/src/main/resources/wsdl/pcssCivil.wsdl</wsdl>
```

Comment:
BASE_DIR for an Env var?

Reply:
Not sure if standards for java are different for that...just asking what the norm for this is.

Reply:
This is a maven var I think, same as ${project.basedir} ?

Reply:
sounds good

Reply:
it is ;-)